src/Components/Tab.tsx

```diff
@@ -19,7 +19,8 @@ export default class HeadlessTab extends React.Component<ITabProps, {}> {
 
     this.headlessTab = buildTab(headlessEngine, {
       options: {
-        expression: this.props.expression
+        expression: this.props.expression,
+        id: this.props.id
```

Comment:
When updated to 0.16.2 I was receiving warning: The #id option on the Tab controller will be required in the future. Please specify it.